DataFormats/simulation/include/SimulationDataFormat/MCTrack.h

```diff
@@ -255,9 +255,8 @@ class MCTrackT
     struct {
       int storage : 1;  // encoding whether to store this track to the output
       unsigned int process : 6; // encoding process that created this track (enough to store TMCProcess from ROOT)
-      int hitmask : 21; // encoding hits per detector
-      int reserved1 : 1; // bit reserved for possible future purposes
-      int reserved2 : 1; // bit reserved for possible future purposes
+      int hitmask : 22;         // encoding hits per detector
+      int reserved1 : 1;        // bit reserved for possible future purposes
```

Comment:
ok. we'll soon be running out of bits and we would soon need a more generic solution...